File: testcases/paper/Variable alias/Variable.cpp
class Variable {
    int i, j;
    int& k = i;

    int method() {
        return i;
    }

    int method2() {
        j = k;
        return j;
    }
};

// Variable aliases don't count: LCOM4 = 2
// Variable aliases count:       LCOM4 = 1

File: testcases/paper/Variable alias/Variable.go
package Variable

type Variable struct {
    i,j int
    k * int
}

func (c Variable) method() {
    c.i = 1
}

func (c Variable) method2() {
    c.k = &c.i
    c.j = *v.k
}

// Variable aliases don't count: LCOM4 = 2
// Variable aliases count:       LCOM4 = 1